target/linux/ramips/dts/mt7628an_tplink_re205-v3.dts

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an_tplink_re205.dtsi"
```

Comment:
Why do you add a separate dtsi file?
If there is only one device using the `mt7628an_tplink_re205.dtsi` file, I would prefer if you incldue the code in the dts file. 

Do you plan to add support for more similar devices?

Reply:
I just tried to copy the style of the RE200 integration.

I removed the `mt7628an_tplink_re205.dtsi` file. As the hardware is almost identical I use the `mt7628an_tplink_re200.dtsi` instead now.